src/js/actions/transform.js

```diff
@@ -1119,9 +1120,9 @@ define(function (require, exports) {
                     nextDoc = appStore.getCurrentDocument();
                     this.flux.actions.layers.resetBounds(nextDoc, nextDoc.layers.allSelected);
                 });
-        }.bind(this);
+        }.bind(this), this);
 
-        _moveListener = function () {
+        _moveListener = synchronization.debounce(function () {
```

Comment:
This is somewhat concerning to me on the history front.  I would have thought that each "move" event corresponds neatly to a history state.  This debouncing will collapse them into a single one, right?  This may get smoothed over by the fact that we are not debouncing the history state listener ... but still, food for thought.